.github/workflows/wireless-performance-autotest.yml

```
@@ -240,13 +246,27 @@
           cat > "$FILENAME" <<- EOT
           ## Devices Under Tests
 
-          This section presents the performance test results, including key metrics and technical details from the test execution.
+          This section presents the performance test results, including key metrics and technical details from the test execution.  
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)